### PR TITLE
Update contribution guide

### PR DESCRIPTION
**Pull request by melnikaite**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -16,26 +16,44 @@ As a contributor, you are expected to fork this repository, work on your own for
...
+    git remote add upstream https://github.com/OpenZeppelin/openzeppelin-sdk.git
+    git fetch upstream
+    git pull --rebase upstream master
+    ```
+
+2. Bootstrap SDK
+    ```
+    npm install
+    npx lerna bootstrap
+    openzeppelin-sdk/packages/cli$ npm link && npm run prepare
```
Comment (jcarpanelli) [inline CONTRIBUTING.md]: This line and #31 are not necessary, as `npx lerna bootstrap` will take care of linking everything and running `npm run prepare` afterwards!

File: packages/cli/test/commands/init.test.js [hunk 2]
```diff
@@ -9,9 +9,9 @@ describe('init command', function() {
   itShouldParse(
     'should call init script with name, version and dependencies',
     'init',
-    'zos init MyApp 0.2.0 --force --no-install --link mock-stdlib@1.1.0,mock-stdlib2@1.2.0',
+    'zos init MyApp 0.2.0 --force --no-install --link mock-stdlib@1.1.0,mock-stdlib-2@1.2.0',
```
Comment (jcarpanelli) [inline packages/cli/test/commands/init.test.js]: Good catch! `mock-stdlib2` doesn't even exist. Tests were passing though, as this test is only asserting that the parameters sent through the `cli` are the same as the ones received by the scripts.

File: CONTRIBUTING.md (hunk 1 above)
Comment (jcarpanelli) [inline CONTRIBUTING.md]:
```suggestion
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -16,26 +16,44 @@ As a contributor, you are expected to fork this repository, work on your own for
...
+    openzeppelin-sdk/packages/lib$ npm link && npm run prepare
```
Comment (jcarpanelli) [inline CONTRIBUTING.md]:
```suggestion
```

**Comment by ylv-io** [comment]
Hey @melnikaite! Thanks for contribution! 🚀

**Pull request merged**